src/main/app/claims/models/claimant-response/court-determination/decisionType.ts

```diff
@@ -2,5 +2,6 @@ export enum DecisionType {
   CLAIMANT_IN_FAVOUR_OF_DEFENDANT = 'CLAIMANT_IN_FAVOUR_OF_DEFENDANT',
```

Comment:
we now have two of these. Please remove one and update the references to the other one.

Reply:
Ok